IFC4x3/Properties/b/BreakingCapacity_0SRRZKplf9oAKBz3w0a2po/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BreakingCapacity_0SRRZKplf9oAKBz3w0a2po" Name="BreakingCapacity" UniqueId="1c6db8d4-cefa-49c8-a50b-f43e80902cf2" PrimaryDataType="IfcReal" />
```

Comment:
Is the PrimaryDataType okay here?

Reply:
Change to Capacitance. Commit : b5e658cad21e4944423ed95d945268c722299c68